conf/application.conf

```diff
@@ -161,7 +161,7 @@ serviceUrls {
   create-compile-event-report-summary = "/pension-online/event-reports/pods/%s"
   compile-event1-report = "/pension-online/event1-report/pods/%s"
   compile-member-event-report = "/pension-online/member-event-reports/%s"
-  submit-event-20A-declaration-report = "/pension-online/event-20a-declaration-report/%s"
+  submit-event-20A-declaration-report = "/pension-online/event-20a-declaration-reports/%s"
```

Comment:
shouldn't the url be like below ?
/pension-online/event20a-declaration-reports/{pstr}':

Difference is :
between event20a and event-20a